source/npm/publish.js

```diff
@@ -1,9 +1,5 @@
-import {execa} from 'execa';
-import {from, catchError} from 'rxjs';
-import handleNpmError from './handle-npm-error.js';
-
-export const getPackagePublishArguments = (options, isYarnBerry) => {
-	const args = isYarnBerry ? ['npm', 'publish'] : ['publish'];
```

Comment:
I don't this can have been ever working - with yarn berry it seems like it'd run `npm npm publish`???

Reply:
It runs yarn npm publish, which is needed for yarn berry because it doesnt have the publish command afaik